README.md

```diff
@@ -1 +1,43 @@
-# Math-Magicians
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math-Magicians
+
+## Description
+
+> Math-Magicians is an oline web calculator built with React Library .
+
+## Built With
+- React 
+- VCS: GitBash
+- Following github flows standards.
+
+## Live demo
+
+[Coming soon]()
+
+## Authors
+
+👤 **Tarik Bouari**
+
+- GitHub: [@tarikbouari](https://github.com/tarikbouari)
+- Twitter: [@tarikbouari](https://twitter.com/TarikBouari)
+- LinkedIn: [tarikbouari](https://www.linkedin.com/in/tarik-bouari-44b7191a6/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Linters Article: Setup Linters
+- Icons resource: [Iconify](https://iconify.design/cons8)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed..
```

Comment:
- [OPTIONAL] the link of the MIT.md is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯